src/app/modules/work-time/work-time.component.ts

```diff
@@ -190,5 +195,16 @@ export class WorkTimeComponent implements ControlValueAccessor, Validator {
     registerOnTouched(fn: any): void {
     }
 
+    emitUpdatedEvent(){
+      this.workTimesUpdated.emit('updated');
+    }
+
+    getDayFormat(day){
```

Comment:
I noted above about saving variants in an object. But if you wan use this function, you can rewrite it shorter. Example:
```
getDayFormat(day: string): string{
  return this.longNames ? day : day.substring(0, 2);
}
```